content/events/proof-of-space-days-ethcc.md

```diff
@@ -0,0 +1,28 @@
+---
+eventName: "Proof of Space Days" #the name of your track or event and its mandatory
+
+eventDescription: "Proof of Space days is a gathering of cryptographers and Web3 researchers to share knowledge on Proof of Space topics. We'll have talks and workshops to collaborate, share ideas and onboard new researchers into this field." #short description of your track or event limiting to 100-150 characters
+
+#topics will show up as labels on the event card.Only a maximum of 4 tags will be displayed on the event card. Some referneces for topics - Blockchain, Web3, Cryptocurrency, Tech Taks,Workshop etc.
+
+eventTopic: 
+  - "Cryptography"
+  - "Web3 Research" 
+
+eventType: "Conference" #Type of the event: Please choose one among the below options or just leave it blank
+
+location: "Paris, France" or "Location TBD" #Specify the location of the event.If you aren't sure about the location then mention "Location TBD"
```

Comment:
@laeccles Please remove the comments as its better suggested. 

Also update the location value to either one of the options: 'Paris, France' or 'Location TBD'

Reply:
I @WinsCodes I have updated this pull request.